lib/components/artwork_grids/generic_grid.js

```diff
@@ -129,6 +129,8 @@ const styles = StyleSheet.create({
 })
 
 const GenericArtworksGridContainer = Relay.createContainer(GenericArtworksGrid, {
+  shouldComponentUpdate: () => true,
```

Comment:
I've been looking at this problem and reading docs for too long so I'm excited for fresh eyes this week.